kernel/arch/dreamcast/kernel/init.c

```diff
@@ -22,17 +22,15 @@
 
 extern int _bss_start, end;
 
-void _atexit_call_all();
-
 /* ctor/dtor stuff from libgcc. */
 #if __GNUC__ == 4
 #define _init init
 #define _fini fini
 #endif
 
-void _init(void);
-void _fini(void);
-void __verify_newlib_patch();
+extern void _init(void);
+extern void _fini(void);
```

Comment:
BlueCrab, we thought ti would be a good idea to mark everything that's pulled in externally from newlib as explicitly "extern," to make it clearer the symbols are not part of KOS.